packages/database/src/core/modelClasses/Country.js

```diff
@@ -12,6 +14,12 @@ export class CountryRecord extends DatabaseRecord {
 }
 
 export class CountryModel extends DatabaseModel {
+  static syncDirection = SyncDirections.PULL_FROM_CENTRAL;
```

Comment:
Do we need `Country`? These specific entity type tables have been pretty much replaced with `Entity`, hopefully we don't have any call for them on DataTrak!